actions/global_actions.jsx

```diff
@@ -287,7 +282,7 @@ async function getTeamRedirectChannelIfIsAccesible(user, team) {
     let teamChannels = getChannelsNameMapInTeam(state, team.id);
     if (!teamChannels || Object.keys(teamChannels).length === 0) {
         // This should be executed in pretty limited scenarios (empty teams)
-        await dispatch(fetchMyChannelsAndMembers(team.id)); // eslint-disable-line no-await-in-loop
+        await fetchMyChannelsAndMembers(team.id)(dispatch, getState); // eslint-disable-line no-await-in-loop
```

Comment:
@jespino made this change. Is there a difference between the two syntaxes?

Reply:
we can remove passing `getState` as part of arguments here in this case and it also something we normally use as a convention. Most of the ones AFAIK existed because of some legacy code